postreise/plot/canvas.py

```diff
@@ -0,0 +1,60 @@
+from bokeh.plotting import figure
+
+
+def create_map_canvas(figsize=None, x_range=None, y_range=None):
+    """Create canvas for maps.
+
+    :param tuple figsize: size of the bokeh figure (in pixels).
+    :param tuple x_range: x range to zoom plot to (EPSG:3857).
+    :param tuple y_range: y range to zoom plot to (EPSG:3857).
+    :return: (*bokeh.plotting.figure*) -- empty canvas.
+    :raises TypeError:
+        if ``figsize`` is not a tuple and elements are not int.
+        if ``x_range`` or``y_range`` are not tuple and elements are not int/float.
+    :raises ValueError:
+        if ``figsize``, ``x_range`` or``y_range`` don't have exactly two elements.
+        if elements of ``figsize`` are negative.
+        if first element of ``x_range`` or``y_range`` is greater than the second one.
+    """
+    if figsize is None:
+        figsize = (1400, 800)
+
+    for var, label in [
+        (figsize, "figsize"),
+        (x_range, "x_range"),
+        (y_range, "y_range"),
```

Comment:
Maybe we can use `eval` here or it is something not recommended.

Reply:
How would you use it?

Reply:
```python
for label in ["figsize", "x_range", "y_range"]:
    var = eval(label)
...
```

Reply:
Done